db.go

```diff
@@ -613,10 +616,6 @@ func OverlappingBlocks(bm []BlockMeta) Overlaps {
 	if len(bm) <= 1 {
 		return nil
 	}
-	sort.Slice(bm, func(i, j int) bool {
```

Comment:
Why are you removing this?

Reply:
This was mutating the input slice in a function that should do a read-only check. Thus current master actually has a regression. The sort is moved properly into the reload function.

Reply:
This is fixed and tested in master via #335 so no longer relevant for this PR.